src/core/commands.ts

```diff
@@ -5,6 +5,26 @@ import { getSurroundingMacroRange, stripText } from '../utils/utils'
 
 const logger = lw.log('Commander')
 
+export async function acquireHostPort() {
+    logger.log('ACQUIREHOSTPORT command invoked.')
+    if (lw.liveshare.isGuest) {
+        await lw.liveshare.getHostServerPort(true)
+    }
+    else {
+        void vscode.window.showInformationMessage('This command only works for guests in a Live Share session.')
+    }
+}
+
+export async function shareHostPort() {
+    logger.log('SHAREHOSTPORT command invoked.')
+    if (lw.liveshare.isHost) {
+        await lw.liveshare.shareServer()
+    }
+    else {
+        void vscode.window.showInformationMessage('This command only works for a host in a Live Share session.')
+    }
+}
+
```

Comment:
I'm thinking of merging these two commands to one but haven't come up with a good name for it.

Reply:
How about "Share/Acquire port"?